roles/openshift_examples/examples-sync.sh

```diff
@@ -5,8 +5,8 @@
 #
 # This script should be run from openshift-ansible/roles/openshift_examples
 
-XPAAS_VERSION=ose-v1.2.0-1
-ORIGIN_VERSION=v1.1
+XPAAS_VERSION=master
```

Comment:
I didn't see another branch.  In any case, I would expect the latest dev version to pull from master and then when it stabilizes we start downloading to a new v1.3 directory (from master).